jetstream/core/orchestrator.py

```diff
@@ -514,6 +514,10 @@ def _prefill_thread(self, idx: int):
           padded_tokens=padded_tokens,
           true_length=true_length,
       )
+      if self._metrics_collector:
```

Comment:
FYI, if you intercept here, it's the TTFT for engine api within JetStream, it's not E2E TTFT

Reply:
Revised this metric to align with existing TTFT calculation